cms/djangoapps/contentstore/helpers.py

```diff
@@ -375,7 +388,9 @@ def import_staged_content_from_user_clipboard(parent_key: UsageKey, request) ->
             parent_xblock,
             store,
             user=request.user,
-            slug_hint=user_clipboard.source_usage_key.block_id,
+            slug_hint=user_clipboard.source_usage_key.block_id
+            if isinstance(user_clipboard.source_usage_key, UsageKey)
+            else None,
```

Comment:
Nit: I just find this very hard to read as the first line seems to stand by itself. Maybe indent it like this?
```suggestion
            slug_hint=(
                user_clipboard.source_usage_key.block_id
                if isinstance(user_clipboard.source_usage_key, UsageKey) else None
            ),
```